.github/workflows/cd.yml

```diff
@@ -36,6 +33,30 @@ jobs:
       - name: Build binary wheel and source tarball
         run: python3 -m build --sdist --wheel --outdir dist/ .
 
+      - name: Store build artifacts
+        uses: actions/upload-artifact@83fd05a356d7e2593de66fc9913b3002723633cb
```

Comment:
Note: If we want to further reduce the TCB we could record the digest of the generated artefacts, store that in an output, and compare it when retrieving the artefacts from the file store. This would remove the need to trust the artefact storage.